.github/workflows/create-channel.yml

```diff
@@ -70,6 +70,7 @@ jobs:
         - al2012-x64
         - al2-x64
         - ubuntu-16-x64
+        - debian-stretch-armv7hf
```

Comment:
the "hf" stands for "hard float? right?
is this docker image guaranteed to run with that exact emulation?

Reply:
The base docker images is just labeled `arm32v7`

Reply:
Raspberry Pi is armv7l or armv6l

Reply:
ugh. I did `apt show --installed` on my Raspberry Pi 3B+ and noticed most packages are labeled `armhf`

I don't understand any of this stuff

Reply:
> the "hf" stands for "hard float? right?
> is this docker image guaranteed to run with that exact emulation?

Yes



> Raspberry Pi is armv7l or armv6l

We can add an armv6l separately